pkg/controller/volume/reconciler/reconciler_test.go

```diff
@@ -59,10 +60,10 @@ func Test_Run_Positive_OneDesiredVolumeAttach(t *testing.T) {
 	asw := cache.NewActualStateOfWorld(volumePluginMgr)
 	ad := attacherdetacher.NewAttacherDetacher(volumePluginMgr)
 	reconciler := NewReconciler(
-		reconcilerLoopPeriod, maxSafeToDetachDuration, dsw, asw, ad)
+		reconcilerLoopPeriod, maxWaitForUnmountDuration, dsw, asw, ad)
 	podName := "pod-name"
-	volumeName := "volume-name"
-	volumeSpec := controllervolumetesting.GetTestVolumeSpec(volumeName, volumeName)
+	volumeName := api.UniqueDeviceName("volume-name")
+	volumeSpec := controllervolumetesting.GetTestVolumeSpec(string(volumeName), volumeName)
```

Comment:
I hear ya, I know it's ugly. But the actual field being populated is a string. These tests happen to use the same value as `UniqueDeviceName` but the actual value is a string. Modifying the function method will make the code prettier, but the caller should be explicitly aware of what they are passing in and be forced to do the conversion (if they are lazy like me and not creating unique string for this value).